### PR TITLE
fixed the reader view navbar color  when in lightmode

### LINKED ISSUE #42: Sidebar is unreadable in Light Mode
The Sidebar in the Reading view does not change the color when the theme is switched to light mode.
Sidebar and the Text inside the Sidebar are both black in color which makes anything written on it illegible

### PR DESCRIPTION
**Pull request by abhijeetChawla**
Closes #42 
The Navbar in reader view is now adopting background colors to light mode

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@ node_modules/
 .vscode
 .env
 
-build/*
\ No newline at end of file
+build/*
+package-lock.json
```
Comment (AriaMoradi) [inline .gitignore:7]: Revert this change, ref: https://dev.to/adamklein/package-lock-json-in-git-or-not-50l5

File: src/components/navbar/ReaderNavBar.tsx [hunk 2]
```diff
@@ -46,10 +46,13 @@ const useStyles = (settings: IReaderSettings) => makeStyles({
...
         overflowY: 'auto',
-        backgroundColor: '#0a0b0b',
+        backgroundColor: theme.palette.mode === 'dark' ? '#0a0b0b' : '#fff',
 
         '& header': {
-            backgroundColor: '#363b3d',
+            backgroundColor:
+            // the shade of white is from the main navbar color
+            // it would be better if we could access this color from the mui theme
+            theme.palette.mode === 'dark' ? '#363b3d' : '#f5f5f5',
```
Comment (AriaMoradi) [inline src/components/navbar/ReaderNavBar.tsx:55]: You can, from the theme object itself: https://mui.com/customization/default-theme/

**Review by AriaMoradi — CHANGES_REQUESTED** [review]

File: src/components/navbar/ReaderNavBar.tsx (hunk 2 above)
Comment (abhijeetChawla) [reply src/components/navbar/ReaderNavBar.tsx:55]: I can't seem to find the exact color as the one used in the Navbar, I'll just use one of the Greys for now

**Comment by abhijeetChawla** [comment]
I made the changes requested

**Review by AriaMoradi — CHANGES_REQUESTED** [review]
This project uses yarn to do package management, you can use npm for your own clone(though not recommended), but don't include the generated `package-lock.json` in your pull request.

File: src/components/navbar/ReaderNavBar.tsx [hunk 3]
```diff
@@ -46,10 +46,11 @@ const useStyles = (settings: IReaderSettings) => makeStyles({
...
         height: '100vh',
         overflowY: 'auto',
-        backgroundColor: '#0a0b0b',
+        backgroundColor: theme.palette.mode === 'dark' ? '#0a0b0b' : 'white',
 
         '& header': {
-            backgroundColor: '#363b3d',
+            backgroundColor:
+            theme.palette.mode === 'dark' ? '#363b3d' : theme.palette.grey[100],
             display: 'flex',
```
[the remark this answers is not in the document]
Comment (voltrare) [reply src/components/navbar/ReaderNavBar.tsx:54]: https://next.material-ui.com/customization/dark-mode/
I used this for the chapter hover thing
the `#0a0b0b` might be comparable to the  `palette.background.default`

File: .gitignore [hunk 4]
```diff
@@ -3,4 +3,4 @@ node_modules/
 .vscode
 .env
 
-build/*
\ No newline at end of file
+build/*
```
Comment (AriaMoradi) [inline .gitignore:6]: Revert changes on this file so the PR is cleaner.

**Review by AriaMoradi — CHANGES_REQUESTED** [review]

File: src/components/navbar/ReaderNavBar.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (abhijeetChawla) [reply src/components/navbar/ReaderNavBar.tsx:54]: So made the navbar use colors from the Theme object. Replaced the colors from the closest counter parts available

File: .gitignore (hunk 4 above)
Comment (abhijeetChawla) [reply .gitignore]: Done

**Review by AriaMoradi — APPROVED** [review]

**Pull request merged**